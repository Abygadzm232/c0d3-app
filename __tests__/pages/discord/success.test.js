import React from 'react'
import ConnectToDiscordSuccess from '../../../pages/discord/success'
import { render } from '@testing-library/react'

describe('connect to Discord success page', () => {
  it('should render page', () => {
    const { container } = render(<ConnectToDiscordSuccess />)
    expect(container).toMatchSnapshot()
  })
})
