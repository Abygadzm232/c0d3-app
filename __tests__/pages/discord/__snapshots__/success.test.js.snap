// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`connect to Discord success page should render page 1`] = `
<div>
  <div
    class="row mt-5"
  >
    <div
      class="card shadow-sm col-sm-8 col-md-7 col-lg-6 col-xl-6 m-auto px-md-5 border-0"
    >
      <div
        class="card-body text-center pt-5 pb-5"
      >
        <h1
          class="card-title h2 font-weight-bold mb-5"
        >
          Success!
        </h1>
        <p
          class="card-text"
        />
        <div
          class="mt-3"
        >
          <p>
            You are now connected to Discord!
          </p>
        </div>
        <button
          class="btn btn-primary btn-lg btn-block mb-3"
          href="/curriculum"
          type="button"
        >
          Continue to Curriculum
        </button>
      </div>
    </div>
  </div>
</div>
`;
